store/driver/txn/union_iter.go

```diff
@@ -102,6 +124,15 @@ func (iter *UnionIter) updateCur() error {
 				break
 			} else {
 				// record from dirty comes first
+				if len(iter.dirtyIt.Value()) == 0 {
+					logutil.BgLogger().Warn("delete a record not exists?",
```

Comment:
Insert some data and delete them, this is possible?

Reply:
Yes, it is possible. However, I copied it from client-go, but did not found this warning before. I think it is because we are using corp API to scan tables, so it didn't happen for dirty transaction. But it will produce the warning if we do not remove it because the UnionIter will also be used in UnionScan, so I think we should still remove it.